### DISCUSSION TITLE
A third theme that has softer contrast then the current day and night themes

### DISCUSSION THREAD

**Discussion by Rainissance** [mozzius/graysky#249] [Feature requests]
I love the app, however I struggle using it as the only themes that are available are too high in contrast for me to use without eye-strain. It is a genuine accessibility issue for me (and I can imagine it's the same for others) and Graysky would benefit from being able to offer a third option that is much softer then the dark mode blackness, i.e Twitter's Dim theme.

**Comment by mozzius** [comment]
Yeah, I'd definitely use a dim theme. I'll look into it 👍

**Comment by kalafalas** [comment]
A dim theme would be fantastic! Something akin to Apollo of old. Also, would it be possible to have transparent tab/nav bars like the rest of the OS? That would be the final visual polish the app needs to really feel native!
